scripts/test

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -x
+
+if [ -f .env ]; then
+  source .env
+fi
+
+echo testing bot template
+
+BOT_NAME="async-box-bot"
+PRJDIR=$(pwd)
+TMPDIR=$(mktemp -d)
+cookiecutter --no-input . -o ${TMPDIR}
+
+cd "${TMPDIR}/${BOT_NAME}" || exit 1
+python -m venv .venv
+source .venv/bin/activate
+pip install poetry
+poetry install
+APP_ENV=test pytest --cov-config=setup.cfg
```

Comment:
Pls use `APP_ENV=$(test pytest --cov-config=setup.cfg)` form.

Reply:
It is not a variable assigning. 
It is just running pytest with APP_ENV set to "test".